vmaas/reposcan/exporter.py

```diff
@@ -748,6 +749,59 @@ def _dump_oval(self, dump):
                 dump.execute("insert into oval_module_test_detail values (?, ?)",
                              (oval_module_test_id, module_stream))
 
+    def _dump_csaf(self, dump):
+        """Select CSAF data"""
+        # Dump data from csaf_product table
+        dump.execute("""CREATE TABLE IF NOT EXISTS csaf_product (
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        cpe_id INTEGER NOT NULL,
+                        package_name_id INTEGER,
+                        package_id INTEGER,
+                        module_stream_id INTEGER
+                    )""")
+        with self._named_cursor() as cursor:
+            cursor.execute("""SELECT *
```

Comment:
Even though we want to include all columns in the dump now, it might be better to list which columns we want to select. We might rename some columns or add more columns to tables in the future. Listing desired columns will guarantee their order. I'd list columns in all selects.